chef/data_bags/crowbar/bc-template-cinder.json

```diff
@@ -19,6 +19,9 @@
       "use_gitbarclamp": true,
       "use_virtualenv": true,
       "use_pip_cache": true,
+      "min_pool_size": 1,
+      "max_pool_size": 15,
```

Comment:
can you use the more production-ready settings that were developed based on the bugreport?